modules/qna/src/backend/utils.ts

```diff
@@ -4,6 +4,18 @@ import _ from 'lodash'
 import { QnaEntry } from './qna'
 import Storage, { NLU_PREFIX } from './storage'
 
+export const QNA_MIN_QUESTIONS = 3
+export const QNA_MIN_ANSWERS = 1
+
+export const isQnaComplete = (qnaEntry: QnaEntry, lang: string): boolean => {
```

Comment:
would be nice to use this function in Front end as well instead of duplicating the code